config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -77,12 +77,12 @@ presubmits:
           --timeout=65m
           "--node-args=--image-config-file=${GOPATH}/src/k8s.io/test-infra/jobs/e2e_node/containerd/containerd-main-presubmit/image-config-presubmit.yaml -node-env=PULL_REFS=$(PULL_REFS)"
 
-  - name: pull-containerd-node-e2e-1-5
+  - name: pull-containerd-node-e2e-1-6
```

Comment:
this job is working with `release-1.25` of Kubernetes (I believe because of cri api version mismatch). Let's switch to master

Reply:
Makes sense, updated

Reply:
maybe just remove this 1-5 job should be fine with the above main, 1.6, 1.7 jobs as we will no longer have a release that is stuck on a back level kubernetes